zitadel/v2/smtp_config/resource.go

```diff
@@ -0,0 +1,49 @@
+package smtp_config
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func GetResource() *schema.Resource {
+	return &schema.Resource{
+		Description: "Resource representing the SMTP configuration of an instance.",
+		Schema: map[string]*schema.Schema{
+			senderAddressVar: {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Address used to send emails.",
+			},
+			senderNameVar: {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Sender name used to send emails.",
+			},
+			tlsVar: {
+				Type:        schema.TypeBool,
+				Required:    true,
```

Comment:
```suggestion
				Required:    false,
```

Reply:
In ZITADEL it's no required, but as the default would be false, I rather define it as required